### ISSUE TITLE
Setting text property on JFXTextField with floating label causes exception

### ISSUE THREAD

**Issue by fehdem** [jfoenixadmin/JFoenix#475]
Hi!
I've got a problem with the JFXTextField. If I make the label float and preset a text I get an exception when loading the FXML. If a remove the text, everything is fine.
Furthermore the same happends sometimes when setting the text at runtime. So far I wasn't able to narrow down the "sometimes" into something tangible.
However, with presetting the text it is clearly reproducible. If I remove the text tag everything goes well.

The FXML:
```
<?import com.jfoenix.controls.JFXTextField?>
<?import javafx.scene.layout.AnchorPane?>
<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/8.0.141" xmlns:fx="http://javafx.com/fxml/1" fx:controller="Playground">
   <children>
      <JFXTextField fx:id="textfield" labelFloat="true" layoutY="54.0" promptText="Hello World" text="..." />
   </children>
</AnchorPane>
```

How I load it:
```
  @Override
  public void start(Stage primaryStage) throws Exception
  {
    Parent root = FXMLLoader.load(getClass().getResource( "Playground.fxml" ) );
    primaryStage.setTitle( "Playground" );
    primaryStage.setScene( new Scene( root ) );
    primaryStage.show();
  }
```

The exception:
```
Exception in Application start method
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(LauncherImpl.java:473)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:372)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at java.base/sun.launcher.LauncherHelper$FXHelper.main(LauncherHelper.java:945)
Caused by: java.lang.RuntimeException: Exception in Application start method
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:973)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:198)
	at java.base/java.lang.Thread.run(Thread.java:844)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private javafx.scene.text.Text javafx.scene.control.skin.TextFieldSkin.promptNode accessible: module javafx.controls does not "opens javafx.scene.control.skin" to unnamed module @590c95cb
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:337)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:281)
	at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:176)
	at java.base/java.lang.reflect.Field.setAccessible(Field.java:170)
	at com.jfoenix.skins.JFXTextFieldSkin.createFloatingLabel(JFXTextFieldSkin.java:425)
	at com.jfoenix.skins.JFXTextFieldSkin.layoutChildren(JFXTextFieldSkin.java:339)
	at javafx.controls/javafx.scene.control.Control.layoutChildren(Control.java:601)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1226)
	at javafx.graphics/javafx.scene.Parent.layout(Parent.java:1233)
	at javafx.graphics/javafx.scene.Scene.doLayoutPass(Scene.java:590)
	at javafx.graphics/javafx.scene.Scene.preferredSize(Scene.java:1773)
	at javafx.graphics/javafx.scene.Scene$2.preferredSize(Scene.java:394)
	at javafx.graphics/com.sun.javafx.scene.SceneHelper.preferredSize(SceneHelper.java:66)
	at javafx.graphics/javafx.stage.Window$12.invalidated(Window.java:1098)
	at javafx.base/javafx.beans.property.BooleanPropertyBase.markInvalid(BooleanPropertyBase.java:110)
	at javafx.base/javafx.beans.property.BooleanPropertyBase.set(BooleanPropertyBase.java:145)
	at javafx.graphics/javafx.stage.Window.setShowing(Window.java:1186)
	at javafx.graphics/javafx.stage.Window.show(Window.java:1201)
	at javafx.graphics/javafx.stage.Stage.show(Stage.java:276)
	at Playground.start(Playground.java:27)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:919)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$11(PlatformImpl.java:449)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$9(PlatformImpl.java:418)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:417)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:189)
	... 1 more
Exception running application Playground
```

**Comment by jfoenixadmin** [comment]
Hello,
It seems you are using Java 9, you need to use JFoenix 9.0.0 instead of 1.9.1. you can download it from <a href="http://www.jfoenix.com/download/jfoenix-9.0.0.jar">here</a>
it will be pushed to maven central soon.

Regards,

**Comment by fehdem** [comment]
Hi!
If I'm not mistaken by accident I'm already using JFoenix 9.0.0. I double checked my configuration and I can't find a failure. 
```
"C:\Program Files\Java\jdk-9\bin\java" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2017.2.5\lib\idea_rt.jar=62626:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2017.2.5\bin" -Dfile.encoding=UTF-8 -classpath C:\Users\Marcus\IdeaProjects\VEBuch\out\production\Playground;C:\Users\Marcus\IdeaProjects\VEBuch\DesktopFrontend\lib\jfoenix-9.0.0.jar Playground
```

Best regards,
Marcus

**Comment by jfoenixadmin** [comment]
Hi,
I fixed the issue and updated JFoenix 9.0.0 on the website, you can download it from <a href="http://www.jfoenix.com/download/jfoenix-9.0.0.jar">here</a>
Thanks for pointing out the issue 👍 
Regards,

**Comment by fehdem** [comment]
Hi,

yes, confirmed. The issue is gone. 
Maybe you should have incremented the version / build number to make the lib distinguishable.
However, thanks for fixing this.

Best regards,
Marcus
